react-scanner.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests, stories, and React Native files.
+	globs: [ '**/!(test|stories)/!(.native.).@(js|ts)?(x)' ],
+	// Exclude any vendor or docs directories
+	exclude: [
+		'bin',
+		'build',
+		'build-module',
+		'docs',
+		'node_modules',
+		'patches',
+		'platform-docs',
+		'test',
+		'tools',
+		'typings',
+		'vendor',
+	],
+	/*
+	 * Filter out any non-component React elements and consider only imports of
+	 * `@wordpress/components` outside of the package.
+	 */
+	importedFrom: '@wordpress/components',
+	// Merge experimental and unstable stats into the canonical component name
+	getComponentName: ( { imported, local } ) => {
+		return ( imported || local )
+			.replace( '__experimental', '' )
+			.replace( '__unstable', '' );
+	},
+	processors: [ [ 'raw-report', { outputTo: './results/gutenberg.json' } ] ],
```

Comment:
Should we use a more specific folder name than `results` ?

Reply:
I think I'm fine with `results` actually. That way we can reuse if we have another tool that generates some statso r something. We're git-ignoring it too, so it's fine. And if we decide to rename, we always can.